src/autogluon/bench/eval/benchmark_context/output_suite_context.py

```diff
@@ -184,6 +184,23 @@ def aggregate_leaderboards(self) -> pd.DataFrame:
         leaderboards_df = pd.concat(leaderboards_list, ignore_index=True)
         return leaderboards_df
 
+    def load_model_failures(self) -> List[pd.DataFrame]:
+        return self._loop_func(func=OutputContext.get_model_failures,
+                               input_list=self.output_contexts,
+                               allow_exception=True)
+
+    def aggregate_model_failures(self) -> pd.DataFrame:
+        model_failures_list = self.load_model_failures()
+        none_count = 0
+        for e in model_failures_list:
+            if e is None:
+                none_count += 1
+        if none_count == len(model_failures_list):
+            return pd.DataFrame()
+        else:
+            model_failures_df = pd.concat(model_failures_list, ignore_index=True)
```

Comment:
What if some of the items in `model_failures_list` are `None`?

Reply:
pd.concat will ignore None elements